src/components/existing-request/existing-request-display.vue

```diff
@@ -249,9 +249,6 @@ export default class ExistingRequestDisplay extends Mixins(
     return Boolean(sessionStorage.getItem(SessionStorageKeys.KeyCloakToken))
   }
 
-  /** This is True while refreshing the NR. (Not used in template at the moment.) */
-  checking = false
```

Comment:
any idea what this was used for previously 😱

Reply:
There are a couple of state variables that are used in the UI. I assume this was one of them that became obsolete either before we started working on this page or in the early days when we refactored this page per UI designs.
I'm not worried about this.